src/pull_request.ts

```diff
@@ -32,6 +32,10 @@ export async function handlePullRequest() {
 
   const octokit = initOctokit(config.githubToken);
 
+  if (shouldIgnorePullRequest(pull_request)) {
+    return;
+  }
```

Comment:
Consider adding a log message before returning to explicitly indicate that the PR review is being skipped. This would improve observability and make it clearer in logs why no review was performed. For example:
```typescript
if (shouldIgnorePullRequest(pull_request)) {
  info('Skipping PR review due to ignore tag in description');
  return;
}
```